draft-irtf-cfrg-opaque.md

```diff
@@ -627,17 +814,13 @@ Steps:
 ### FinalizeRequest {#finalize-request}
 
 ~~~
-FinalizeRequest(password, creds, blind, response)
-
-Parameters:
-- params, the MHF parameters established out of band
-- mode, the InnerEnvelope mode
-- Nx, the output size of the Extract function
+FinalizeRequest(client_private_key, password, blind, response)
```

Comment:
In external mode, implementations can be free to add the public key here. If added, then the public key doesn't have to be recovered in BuildInnerEnvelope() and the function must be adapted accordingly.

cc @chris-wood